### PR TITLE
update working script for validation

### PR DESCRIPTION
**Pull request by thuanguy**

### CODE REVIEW

**Comment by gboeing** [comment]
In general you don't have to create a new pull request, you can just reopen the existing one (#24) next time. But for simplicity let's just work here in this one now. Let me know if this is ready for review again prior to merge.

File: process/configuration/hanoi.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "geopackagePath": "hanoi_vn_2019_1600m_buffer.gpkg",
     "graphmlName": "hanoi_vn_2019_10000m_pedestrian_osm_20190902.graphml",
     "graphmlProj_name": "hanoi_vn_2019_10000m_pedestrian_osm_20190902_proj.graphml",
-    "folder": "data/input",
```
Comment (gboeing) [inline process/configuration/hanoi.json:7]: This PR isn't intended to be updating the process folder's config scripts right? It's just for the validation work? If so, please reset any non-validation files and re-push, so they're not in this PR.

**Review by gboeing — CHANGES_REQUESTED** [review]

File: process/configuration/hanoi.json (hunk 1 above)
Comment (gboeing) [reply process/configuration/hanoi.json:7]: @thuanguy don't delete the files (hanoi, bern, belfast) then commit the deletion. Just reset the files back to the point at which you branched.

Comment (thuanguy) [reply process/configuration/hanoi.json:7]: I couldn't find the place to reset the file.

Comment (gboeing) [reply process/configuration/hanoi.json:7]: See this for example: https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

You'll want to return any files you're not actually updating in this PR back to their state at which you branched.

**Pull request closed without merging**